lib/dry/web/roda/templates/Rakefile.tt

```diff
@@ -32,6 +32,16 @@ namespace :db do
     puts "Current schema version: #{version}"
   end
 
+  desc "Creates database"
+  task :create do
+    if system("which createdb", out: File::NULL)
+      <%= config[:camel_cased_app_name] %>::Container.boot :settings
+      settings = <%= config[:camel_cased_app_name] %>::Container["settings"]
+      database_name = settings.database_url.split('/').last
+      `createdb #{Shellwords.escape(database_name)}`
+    end
```

Comment:
And I reckon it might be good to have an `else` branch here where we print a notice about requiring postgres to be installed (and perhaps `exit` with a non-zero exit code too).